src/components/organisms/AdminVenueView/components/EventCard/EventCard.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import classNames from "classnames";
+import { sortBy } from "lodash";
+
+import { VenueEvent } from "types/venues";
+
+import { isEventLive } from "utils/event";
+import { WithId, WithVenueId } from "utils/id";
+import { formatTimeLocalised } from "utils/time";
+import { adminNGVenueUrl } from "utils/url";
+
+import { useVenueId } from "hooks/useVenueId";
+
+import { AdminVenueTab } from "components/organisms/AdminVenueView/AdminVenueView";
+
+import "./EventCard.scss";
+
+interface EventCardProps {
+  events?: WithVenueId<WithId<VenueEvent>>[];
+}
+
+export const EventCard: React.FC<EventCardProps> = ({ events }) => {
+  const venueId = useVenueId();
+  const [nextEvent, followingEvent] = sortBy(events ?? [], "start_utc_seconds");
```

Comment:
there should be a sort in the `utils/event`: eventTimeComparator. Maybe you could use that as well.